app/views/registration/register.jade

```diff
@@ -54,8 +54,8 @@ block body
             label(class="text-"+controlGroupClass)
               input(type="checkbox", name="terms", tabindex=tabIndex++)
               |  I confirm that the information I have entered above is not
-              |  fraudulent, and that I agree to be bound by the 
+              |  fraudulent, and that I agree to be bound by the
```

Comment:
Be very very careful of trimming trailing whitespace in Jade - in this case, it's omission will result in the word "the" merging with the link text from the next line.